src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -168,6 +170,17 @@ def __writeSub(self, executable, nJobs, location, processors):
 
         executable = os.path.join(self.workingDirectory, executable)
 
+        useCredentials = "use_x509userproxy = true"
+        if tokenFile:
+            useCredentials = (
+                """
+use_x509userproxy = true
```

Comment:
This is now always present, is that intentional? if yes, why not leave it in the default?

Reply:
Yes, we always ask to pass X509 proxy to the job environment, even if we submit with tokens.